transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -289,26 +320,10 @@ static jobject _recvFrom(JNIEnv* env, jint fd, void* buffer, jint pos, jint limi
     return createDatagramSocketAddress(env, &addr, res);
 }
 
-static void netty_unix_socket_optionHandleError(JNIEnv* env, int err, char* method) {
-    if (err == EBADF) {
```

Comment:
IIUC the static methods have to move up to avoid their first use case or we need to declare the method before the first use.